src/repr/src/global_id.rs

```diff
@@ -92,6 +99,7 @@ impl FromStr for GlobalId {
         let val: u64 = s[1..].parse()?;
         match s.chars().next().unwrap() {
             's' => Ok(GlobalId::System(val)),
+            'i' => Ok(GlobalId::IntrospectionSourceIndex(val)),
```

Comment:
Should we make this prefix `si`, to make it clear that this is still a system ID? Perhaps a question to bring before the SQL council, if it wasn't already.

Reply:
I brought it up with the council: https://materializeinc.slack.com/archives/C063H5S7NKE/p1732719347690519

Reply:
I've pushed a commit to switch this to `si`.